### ISSUE TITLE
Need more updates!

### ISSUE THREAD

**Issue by manel00** [rutube/RutubeFramework_Android#1]
Hello everyone!

Thanks to create Framework for Android, but it doesn't work... needs to have a look...

Thanks in advance!

Kind Regards!

**Comment by Juraldinio** [comment]
Yes, we change concept and prepare android framework reincarnation.
I hope it will take about 2 weeks.

**Comment by manel00** [comment]
Are you still giving support? Thanks
